modules/hcp-virt-hc-base-domain.adoc

```diff
@@ -66,7 +66,7 @@ console                                    4.x.0     False       False         F
 ingress                                    4.x.0     True        False         True       28m     The "default" ingress controller reports Degraded=True: DegradedConditions: One or more other status conditions indicate a degraded state: CanaryChecksSucceeding=False (CanaryChecksRepetitiveFailures: Canary route checks for the default ingress controller are failing)
 ----
 +
-Replace `4.x.0` with the supported {product-title} version that you want to use.
+Replace `<4.x.0>` with the supported {product-title} version that you want to use.
```

Comment:
In the example output, place `4.x.0` in angle brackets to match the annotation text.